core/shared/src/main/scala/zio/prelude/Debug.scala

```diff
@@ -415,6 +415,12 @@ object Debug extends DebugVersionSpecific {
   implicit def NonEmptyChunkDebug[A: Debug]: Debug[NonEmptyChunk[A]] =
     nonEmptyChunk => Repr.VConstructor(List("zio"), "NonEmptyChunk", nonEmptyChunk.map(_.debug).toList)
 
+  /**
+   * Derives a `Debug[Set[A]]` given a `Debug[A]`.
+   */
+  implicit def SetDebug[A: Debug]: Debug[Set[A]] =
```

Comment:
Great to add these instances! Can we add them to the tests too?

Reply:
Sure. Let me do it.